<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>/*  Copyright (C) 2009 Klar√§lvdalens Datakonsult AB, a KDAB Group company, info@kdab.net
  Copyright (c) 2009 Andras Mantia &lt;andras@kdab.net&gt;

  This program is free software; you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation; either version 2 of the License, or
  (at your option) any later version.

  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  GNU General Public License for more details.

  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
*/</author>
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_showColorBar">
     <property name="text">
      <string>Show &amp;HTML status bar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_showSpamStatus">
     <property name="text">
      <string>Show spam status &amp;in fancy headers</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_ShowEmoticons">
     <property name="text">
      <string>Replace smileys &amp;by emoticons</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_ShrinkQuotes">
     <property name="text">
      <string>Reduce font size for &amp;quoted text</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="kcfg_ShowExpandQuotesMark">
     <property name="text">
      <string>Show &amp;expand/collapse quote marks</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Au&amp;tomatic collapse level:</string>
     </property>
     <property name="buddy">
      <cstring>kcfg_CollapseQuoteLevelSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="KIntSpinBox" name="kcfg_CollapseQuoteLevelSpin"/>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>273</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fallback character e&amp;ncoding:</string>
       </property>
       <property name="buddy">
        <cstring>kcfg_FallbackCharacterEncoding</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="kcfg_FallbackCharacterEncoding">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Override character encoding:</string>
       </property>
       <property name="buddy">
        <cstring>kcfg_OverrideCharacterEncoding</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="kcfg_OverrideCharacterEncoding">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
